templates/default/event/eventinfoshort.html

```diff
@@ -184,35 +185,37 @@
             <A href="?m=customerinfo&id={$event.customerid}">{$event.customername}</A>
         </TD>
     </TR>
+    {/if}
+    {if $event.netnode_name || $event.customerlocation}
     <TR>
         <TD>
-            {icon name="location"}
+            {icon name="location" class="fa-fw"}
         </TD>
         <TD class="bold">
             {trans("Location:")}
         </TD>
         <TD class="nobr">
         {if $event.netnode_name}
-            {icon name="netnode"} <a href="?m=netnodeinfo&id={$event.netnode_id}">{$event.netnode_name}</a>
-            {if !empty($event.netnode_location)}<br><i class="lms-ui-icon-location"></i> {$event.netnode_location}{/if}
+            {icon name="netnode" class="fa-fw"}<a href="?m=netnodeinfo&id={$event.netnode_id}">{$event.netnode_name}</a>
+            {if !empty($event.netnode_location)}<br>{$event.netnode_location}{/if}
         {else}
             {if $event.customername}
-                {icon name="customer"} <A href="?m=customerinfo&amp;id={$event.customerid}">{$event.customername|truncate:"30":"...":true}</A><br>
```

Comment:
Nie będzie możliwości przejścia z tego miejsca do klienta?

Reply:
A po co to powielać? Jest to samo wyżej...
https://github.com/lmsgit/lms/pull/1804/files/e9c9d287ae62ef32396fa5596e3530b1c5765fa0#diff-8f6a04cf86797a4a37bad42df155e1f0R185